src/components/HelpPopover/HelpPopover.tsx

```diff
@@ -18,7 +13,7 @@ export type HelpPopoverProps = DistributiveOmit<PopoverProps, 'children'>;
 
 export function HelpPopover(props: HelpPopoverProps) {
     return (
-        <Popover offset={{left: 4}} {...props} className={b(null, props.className)}>
```

Comment:
Don't remove className, only remove import

Reply:
I see it as a convention that the component should have its own class name.

Reply:
But we don't need any css rules for this component. why do we have add class with empty content?

Reply:
@ValeraS, can you take a look please?

Reply:
Returned the className for better readability of the components in the devTools